webui/src/js/views/domain-design-view.html

```diff
@@ -468,6 +468,10 @@ <h6 class="wkt-subheading"><oj-bind-text value="[[labelMapper('domain-server-tit
                      validators="[[project.k8sDomain.memoryLimit.validators()]]"
                      help.instruction="[[labelMapper('memory-limit-help')]]">
       </oj-input-text>
+      <oj-input-text label-hint="[[labelMapper('node-selector-label')]]"
+                     value="{{project.k8sDomain.nodeSelector.observable}}" class="editable"
+                     help.instruction="[[labelMapper('node-selector-help')]]">
+      </oj-input-text>
```

Comment:
How is this supposed to work?

I would have assumed that the user can tell you both the label name and the label value that they want to use as the node selector?  It seems like you are assuming a label name but it isn't clear what that is...